CONTRIBUTING.md

```diff
@@ -0,0 +1,73 @@
+# Contributing to Auto_Jobs_Applier_AIHawk
+
+Thank you for your interest in contributing to Auto_Jobs_Applier_AIHawk. This document provides guidelines for contributing to the project.
+
+## Bug Reports
+
+When submitting a bug report, please include:
+
+- A clear, descriptive title prefixed with [BUG]
+- Steps to reproduce the issue
+- Expected behavior
+- Actual behavior
+- Any error messages or screenshots
+- Your environment details (OS, Python version, etc.)
+
+## Feature Requests
+
+For feature requests, please:
+
+- Prefix the title with [FEATURE]
+- Include a feature summary
+- Provide detailed feature description
+- Explain your motivation for the feature
+- List any alternatives you've considered
+
+## Pull Request Process
+
+1. Fork the repository
+2. Create a new branch for your feature/fix
+3. Write clear commit messages
+4. Update documentation as needed
+5. Submit a pull request with a clear description
+
+## Code Style Guidelines
+
+- Follow PEP 8 standards for Python code
+- Include docstrings for new functions and classes
+- Add comments for complex logic
+- Maintain consistent naming conventions
```

Comment:
can you add this clean sheet guide

- Follow this clean code sheetsheet - https://www.planetgeek.ch/wp-content/uploads/2014/11/Clean-Code-V2.4.pdf